sympy/utilities/lambdify.py

```diff
@@ -112,13 +166,12 @@ def _import(module, reload="False"):
     Creates a global translation dictionary for module.
 
     The argument module has to be one of the following strings: "math",
-    "mpmath", "numpy", "sympy".
+    "mpmath", "numpy", "sympy", "scipy".
```

Comment:
What order of precedence do you have here? I think it should go along side NumPy.